backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -545,6 +545,41 @@ class TestOperator extends VeloxWholeStageTransformerSuite {
       runQueryAndCompare("SELECT c1, explode(array(c2)) FROM t") {
         checkOperatorMatch[GenerateExecTransformer]
       }
+
+      runQueryAndCompare("SELECT c1, explode(c3) FROM (SELECT c1, array(c2) as c3 FROM t)") {
+        checkOperatorMatch[GenerateExecTransformer]
+      }
+    }
+  }
+
+  test("Add the missing Generate validation check") {
```

Comment:
Nit:
suggested test title:
validation should fail if unsupported expression is used for Generate.

Reply:
Greate